scripts/qa_crowbarsetup.sh

```diff
@@ -2387,7 +2387,7 @@ function custom_configuration()
     ###       So, only edit the proposal file, and NOT the proposal itself
 
     case "$proposal" in
-        keystone|glance|neutron|cinder|swift|nova|horizon|nova_dashboard)
+        keystone|glance|neutron|cinder|swift|nova|horizon|heat|nova_dashboard)
```

Comment:
I would not separate `horizon|nova_dashboard` because are alias for the same thing (there was some renaming)

But basically +1